### DISCUSSION TITLE
Citation_Record and Template Proposal

### DISCUSSION THREAD

**Discussion by Norwegian-Sardines** [FamilySearch/GEDCOM#534] [Ideas]
Here is a rather long proposal for a Shared Citation and Template System.  It probably needs some additional input, so consider it a starting point.

[Proposal - Citation Record.pdf](https://github.com/user-attachments/files/16593451/Proposal.-.Citation.Record.pdf)

**Comment by albertemmerich** [comment]
**@Norwegian-Sardines**
your solution has the same problem as Dave's solution: It uses
``` gedcom
n FIELD <Tag_Name_Text> {1:1}
```
implementing language and user (application-) dependant field names. Again the various elements in todays PAGE payload will not be transferable in between applications, as the payload of FIELD is not standardized.

**Comment by Norwegian-Sardines** [comment]
Citation Standardization comes from the use of a standard citation model such as “Evidence Explained”, “Chicago”, “MLA”, etc.

Language translation of the tag value is only partly necessary when displaying a form that asked for individuals to enter data.  This needs to be done by the application when they except a template for use, This is why the Template has a required description!  

The Citation_Detail can have 100s of various fields, no amount of hard coded tags will be sufficient to support all of the combinations of needed data points.  Your list of `PAGE` tags is too short and incomplete for actual use!
